### PR TITLE
cmd, epochStart, integrationTests: fix failed activation

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
This PR activates the APR changes which failed to activate previously due to some duplicated code.
Duplicated code was removed and rewards computation uses the common handler which is aware of the changed parameters.

### CODE REVIEW

File: epochStart/metachain/rewardsV2.go [hunk 1]
```diff
@@ -56,19 +54,15 @@ func NewRewardsCreatorV2(args RewardsCreatorArgsV2) (*rewardsCreatorV2, error) {
 	if check.IfNil(args.EconomicsDataProvider) {
 		return nil, epochStart.ErrNilEconomicsDataProvider
 	}
-	if args.TopUpGradientPoint.Cmp(zero) < 0 {
+	if check.IfNil(args.RewardsHandler) {
 		return nil, epochStart.ErrInvalidRewardsTopUpGradientPoint
```
Comment (sasurobert) [inline epochStart/metachain/rewardsV2.go:58]: Wrong error message

**Review by sasurobert — COMMENTED** [review]
Can we test by moving the activation period after staking v3.5? With one epoch at least?

File: epochStart/metachain/rewardsV2.go (hunk 1 above)
Comment (AdoAdoAdo) [reply epochStart/metachain/rewardsV2.go]: done

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by mihaieremia — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**